integration_tests/src/main/python/parquet_test.py

```diff
@@ -685,3 +685,48 @@ def test_parquet_reading_from_unaligned_pages_basic_filters_with_nulls(spark_tmp
         assert_gpu_and_cpu_are_equal_collect(
                 lambda spark : spark.read.parquet(data_path).filter(filter_str),
                 all_confs)
+
+
+conf_for_parquet_aggregate_pushdown = {
+    "spark.sql.parquet.aggregatePushdown": "true", 
+    "spark.sql.sources.useV1SourceList": ""
+}
+
+@pytest.mark.skipif(is_before_spark_330(), reason='Aggregate push down on Parquet is a new feature of Spark 330')
+@allow_non_gpu(any = True)
+def test_parquet_scan_without_aggregation_pushdown_not_fallback(spark_tmp_path):
+    """
+    No aggregation will be pushed down in this test, so we should not fallback to CPU
+    """
+    data_path = spark_tmp_path + "/pushdown.parquet"
+
+    def do_parquet_scan(spark):
+        spark.range(10).selectExpr("id", "id % 3 as p").write.partitionBy("p").mode("overwrite").parquet(data_path)
+        df = spark.read.parquet(data_path).selectExpr("Max(p)")
+        return df
+
+    assert_cpu_and_gpu_are_equal_collect_with_capture(
+        do_parquet_scan,
+        exist_classes= "GpuBatchScanExec",
+        non_exist_classes= "BatchScanExec",
+        conf= conf_for_parquet_aggregate_pushdown)
```

Comment:
This should just be `assert_cpu_and_gpu_are_equal_collect`.  We're honestly not that interested in exactly which GPU nodes are involved here, rather that the plan is all on the GPU.  That's what `assert_cpu_and_gpu_are_equal_collect` already checks, once we remove the `@allow_non_gpu` decorator.

Reply:
Thank you! I have updated